homeworks/vlad.makedon_Vladmaked/homework_7/fibonacci.js

```diff
@@ -0,0 +1,10 @@
+function getFibonacciNumbers(rand) {
+    const arrayOfFibonacciNumbers = [0, 1];
+    for (let i = 2; arrayOfFibonacciNumbers[i - 1] + arrayOfFibonacciNumbers[i - 2] < rand; i++) {
```

Comment:
I would keep variable naming a bit shorter, something like `fiboArr` in your case